src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,14 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+interface TodoListProps {
+  todoList: Todo[]
+}
+
+export const TodoList: React.FC<TodoListProps> = ({ todoList }) => (
```

Comment:
I believe this should be named `todos` to make tests pass, because that's how it's named an passed as a prop there
```suggestion
export const TodoList: React.FC<TodoListProps> = ({ todos }) => (
```